storage/series/buffer_test.go

```diff
@@ -225,6 +225,67 @@ func TestBufferDrain(t *testing.T) {
 	assertValuesEqual(t, data[4:], results, opts)
 }
 
+func TestBufferMinMax(t *testing.T) {
```

Comment:
This looks good, do you want to perhaps add a very brief one to verify bootstrapping a block that's already drained returns an error?

Reply:
done